src/Symfony/Bundle/TwigBundle/CacheWarmer/TemplateCacheCacheWarmer.php

```diff
@@ -73,7 +73,7 @@ public function warmUp($cacheDir)
 
         foreach ($templates as $template) {
             try {
-                $twig->loadTemplate($template);
+                $twig->load($template)->unwrap();
```

Comment:
no need to unwrap here (which avoids using an internal method)

Reply:
fixed